### PR TITLE
OmegaT

### PR DESCRIPTION
**Pull request by jbeagley52**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Comment by jbeagley52** [comment]
OmegaT Cask Installer

File: Casks/okapi-apps.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'okapi-apps' do
+  version '0.32'
+  sha256 '5d63f1ef997b05b4faa896a10a2d35ae4998648fada60230e65757a240c62c5c'
+
+  # bintray.com/okapi was verified as official when first introduced to the cask
+  url "http://dl.bintray.com/okapi/Distribution/okapi-apps_cocoa-macosx-x86_64_#{version}.dmg"
+  name 'Okapi-Apps'
+  homepage 'http://okapiframework.org/'
+
+  suite "Okapi_#{version}", target: 'Okapi'
```
Comment (vitorgalvao) [inline Casks/okapi-apps.rb]: [`target:` Should Only Be Used in Select Cases](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/app.md#target-should-only-be-used-in-select-cases)

**Comment by vitorgalvao** [comment]
In the future, please submit separate pull requests for each cask, as more often than not this type of pull request brings problems with one or more of them, and it halts the inclusion of the other ones.

File: Casks/okapi-apps.rb (hunk 1 above)
Comment (neutric) [inline Casks/okapi-apps.rb]: Those changes should probably be done in #30250, which seeks to add Okapi.

**Pull request merged**